src/components/TaxPayer/TaxPayer.tsx

```diff
@@ -0,0 +1,136 @@
+    dispatch(savePrimaryPersonInfo(primaryPerson))
+    onAdvance()
+  }
+
+  let csz
+  if (!isForeignCountry) {
+    csz = (
+      <div>
+        <USStateDropDown
+          label="State"
+          name="address.state"
+          control={control}
+          error={errors.address?.state}
+          required={!isForeignCountry}
+          defaultValue={taxPayer?.primaryPerson?.address.state}
+        />
+        <LabeledInput
+          label="Zip"
+          register={register}
+          error={errors.address?.zip}
+          name="address.zip"
+          patternConfig={Patterns.zip}
+          required={!isForeignCountry}
+          defaultValue={taxPayer?.primaryPerson?.address.zip}
+        />
+      </div>
+    )
+  } else {
+    csz = (
+      <div>
+        <LabeledInput
+          label="Province"
+          name="address.province"
+          register={register}
+          error={errors.address?.province}
+          required={isForeignCountry}
+          defaultValue={taxPayer?.primaryPerson?.address.province}
+        />
+        <LabeledInput
+          name="address.postalCode"
+          label="Postal Code"
+          register={register}
+          error={errors.address?.postalCode}
+          required={isForeignCountry}
+          defaultValue={taxPayer?.primaryPerson?.address.postalCode}
+        />
+        <LabeledInput
+          name="address.foreignCountry"
+          label="Country"
+          register={register}
+          error={errors.address?.foreignCountry}
+          required={isForeignCountry}
+          defaultValue={taxPayer?.primaryPerson?.address.foreignCountry}
+        />
+      </div>
+    )
+  }
+
+  return (
+    <Box display="flex" justifyContent="center">
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Box display="flex" justifyContent="flex-start">
+          <h2>Taxpayer Information</h2>
+        </Box>
+
+        <h4>Primary Taxpayer Information</h4>
+        <PersonFields
+          register={register}
+          errors={errors}
+          defaults={taxPayer?.primaryPerson}
+        />
+        <LabeledInput
+          label="Address"
+          name="address.address"
+          register={register}
+          required={true}
+          error={errors.address?.address}
+          defaultValue={taxPayer?.primaryPerson?.address.address}
+        />
+        <LabeledInput
+          label="Unit No"
+          register={register}
+          name="address.aptNo"
+          required={false}
+          error={errors.address?.aptNo}
+          defaultValue={taxPayer?.primaryPerson?.address.aptNo}
+        />
+        <LabeledInput
+          label="City"
+          register={register}
+          name="address.city"
+          patternConfig={Patterns.name}
+          required={true}
+          error={errors.address?.city}
+          defaultValue={taxPayer?.primaryPerson?.address.city}
+        />
+        <LabeledCheckBox
+          label="Do you have a foreign address?"
```

Comment:
This label should be changed to "check the box below if you have a foreign  address since the options are no longer yes and no

Reply:
good?

Reply:
The alignment is not great, revising.

Reply:
check again

Reply:
The green/grey switch is a cool concept, but I think it's a little distracting, should probably stick with black font for the checkbox instructions. Also, I was thinking about it and the yes/no checkbox text should probably be removed if the label is going to be _Check if you have a foreign address_  since that text combination is a little confusing.

Reply:
Ok, revised to single line, and took out the material ui styled title label thing.